rpc/mocks_test.go

```diff
@@ -46,19 +46,19 @@ func (*mockNet) ConnectedPeers() []string {
 	panic("not implemented")
 }
 
-func (*mockNet) Discover(provides string, searchTime time.Duration) ([]peer.ID, error) {
+func (*mockNet) Discover(_ string, _ time.Duration) ([]peer.ID, error) {
```

Comment:
I had to edit this file for the PR, and it was bothering me that all 3 different ways of dealing with unused method parameters were being mixed in the same file (sometimes even on the same method):
* giving the parameter a name
* omitting the parameter name
* using an underscore for the parameter name

In some places, multiple different ways were being use on the same function declaration.